### ISSUE TITLE
Create function for keeping track of number of connections

### ISSUE THREAD

**Issue by jakobj** [jakobj/nest-simulator#29]
The following piece of code appears in several places in `ConnectionManager` and should be moved to a function

```c++
  if ( num_connections_[ tid ].size() <= syn_id )
  {
    num_connections_[ tid ].resize( syn_id + 1 );
  }
  ++num_connections_[ tid ][ syn_id ];
```

**Comment by jakobj** [comment]
This appears three times, in `connect`, `connect_from_device` and `connect_to_device`. @jarsi, could you work on this? just create a function, e.g., `increase_connection_count( tid, syn_id)` to remove some duplicated code.

**Comment by jarsi** [comment]
Sure, will try to remove some duplications.

Jakob Jordan <notifications@github.com> schrieb am Mi., 4. Apr. 2018, 16:35:

**Comment by jakobj** [comment]
closed by #57
